src/documents/IoCReferences.md

```diff
@@ -73,7 +78,7 @@ References always take the syntactic form `{context-name}.some.path.segments` -
                     <li>
                         <code>&lt;path to member&gt;</code> is an EL path into the referenced component's members.
                         <p>
-                            <em>Note that full IoCSS expressions are not valid in all contexts. They can primarily be used in the <code>target</code> field of the <code>distributeOptions</code> record.</em>
+                            <em>Note that full IoCSS expressions are not valid in all contexts. They can primarily be used in the <code>target</code> field of the <a href="IoCSS.md#distributeoptions-format"><code>distributeOptions</code></a> record.</em>
```

Comment:
We should probably style the notes in the table with the "infusion-docs-note" styling.